providers/evercrypt/EverCrypt.DRBG.fsti

```diff
@@ -0,0 +1,373 @@
+module EverCrypt.DRBG
+
+open FStar.HyperStack.ST
+open Lib.IntTypes
+
+open Spec.Hash.Definitions
+
+module HS = FStar.HyperStack
+module B = LowStar.Buffer
+module S = Spec.HMAC_DRBG
+
+#set-options "--max_ifuel 0 --max_fuel 0"
+
+/// HMAC-DRBG
+///
+/// See 10.1.2 and B.2 of
+/// https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-90Ar1.pdf
+///
+/// This module implements
+/// - HMAC_DRBG_Instantiate_function
+/// - HMAC_DRBG_Reseed_function
+/// - HMAC_DRBG_Generate_function
+/// - HMAC_DRBG_Uninstantiate_function
+///
+/// Internally, it uses Lib.RandomBuffer.System as the Get_entropy_input function,
+/// for instantiation, reseeding, and prediction resistance.
+///
+/// - Supports SHA-1, SHA2-256, SHA2-384 and SHA2-512
+///
+/// - Supports reseeding
+///
+/// - Supports optional personalization_string for instantiation
+///
+/// - Supports optional additional_input for reseeding and generation
+///
+/// - Always provides prediction resistance (i.e. reseeds before generation)
+///
+/// - The internal state is (Key,V,reseed_counter)
+///
+/// - The security_strength is the HMAC-strength of the hash algorithm as per p.54 of
+///   https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-57pt1r4.pdf
+///
+/// - The minimum entropy for instantiation is 3/2 * security_strength.
+///    - entropy_input must have at least security_strength bits.
+///    - nonce must have at least 1/2 security_strength bits.
+///    - entropy_input and nonce can have at most max_length = 2^16 bits.
+///
+/// - At most max_number_of_bits_per_request = 2^16 bits can be generated per request.
+
+
+/// Some duplication from Hacl.HMAC_DRBG because we don't want clients to depend on it
+
+unfold
+let supported_alg = S.supported_alg
+
+//[@ CMacro ]
```

Comment:
Using `@CMacro` breaks F* clients, which still refer to the non-macro definition.

Reply:
have you filed a bug for kremlin on that? I think it's a lack of attributes on assumed vals

Reply:
But here the attribute is attached to a definition. Which assumed `val` you mean?
I didn't look further into it, but I initially thought it was a problem with bundling.

Reply:
Ah I thought you meant clients like miTLS who work against fsti-only. If you have a minimal repro I'm happy to look at it.